PFS-IPMU-00001/startdown.rst

```diff
@@ -0,0 +1,51 @@
+Service cold start and full shutdown procedures
+------
+
+This memo shows ways to start all from cold and to shutdown all, with points 
+in consideration or taken cared of.
+
+Cold start
+======
+
+Start backends first, and end point to external. 
+
+* Power on storage server
+
+  * iSCSI server need to be started from JBOD, and controller follows
+  * Just power on SAS storage box
+
+* Power on VM host for NFS servers (dlb8-vm)
```

Comment:
Before it, you suggested check the all network switches are on and network looks available.

Reply:
added ups/network switch.
LACP is in configuration, so it should be fine.